packages/vscode-extension/src/handlers.ts

```diff
@@ -222,6 +222,7 @@ export async function activate(): Promise<Result<Void, FxError>> {
     await openSampleReadmeHandler();
     await postUpgrade();
     ExtTelemetry.isFromSample = await getIsFromSample();
+    ExtTelemetry.createdFrom = await getCreatedFrom();
```

Comment:
This might be undefined sometimes and we have an extra field in core for isFromSample to avoid the issue.

Reply:
thanks for the advice, updated.